pyproject.toml

```diff
@@ -16,50 +16,46 @@ exclude = '''
 line_length = 88
 profile = "black"
 
-
 [build-system]
-requires = ["setuptools>=61.2"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry_dynamic_versioning.backend"
 
-[project]
+[tool.poetry]
 name = "openshift-python-utilities"
-authors = [
-    {name = "Meni Yakove", email = "myakove@gmail.com"},
-    {name = "Ruth Netser", email = "rnetser@gmail.com"}
-]
+version = "0.0.0"
 description = "A utilities repository for https://github.com/RedHatQE/openshift-python-wrapper"
+authors = ["Meni Yakove <myakove@gmail.com>", "Ruth Netser <rnetser@gmail.com>"]
+readme = "README.md"
+license = "Apache-2.0"
+packages = [{include = "ocp_utilities"}]
+homepage = "https://github.com/RedHatQE/openshift-python-utilities"
+documentation = "https://github.com/RedHatQE/openshift-python-utilities/blob/main/README.md"
+keywords = ["Openshift", "ocp"]
 classifiers = [
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent",
 ]
-requires-python = ">=3.8"
-keywords = ["Openshift"]
-dependencies = [
-    "openshift",
-    "colorlog",
-    "pytest",
-    "pytest-testconfig",
-    "PyYAML",
-    "openshift-python-wrapper-data-collector",
-    "python-simple-logger",
-]
-dynamic = ["version"]
-# ...
-[tool.setuptools.dynamic]
-version = {file = "VERSION"}
 
-[project.readme]
-file = "README.md"
-content-type = "text/markdown"
+[tool.poetry-dynamic-versioning.substitution]
+files = ["VERSION"]
 
-[project.urls]
-Homepage = "https://github.com/RedHatQE/openshift-python-utilities"
+[tool.poetry.urls]
 Download = "https://pypi.org/project/openshift-python-utilities/"
 "Bug Tracker" = "https://github.com/RedHatQE/openshift-python-utilities/issues"
-Documentation = "https://github.com/RedHatQE/openshift-python-utilities/blob/main/README.md"
-
-[tool.setuptools]
-include-package-data = true
 
-[tool.setuptools.packages.find]
-include = ["ocp_utilities"]
+[tool.poetry.dependencies]
+python = "^3.8"
+openshift = "^0.13.2"
+colorlog = "^6.7.0"
+pytest = "^7.4.0"
+pytest-testconfig = "^0.2.0"
+pyyaml = "^6.0.1"
+openshift-python-wrapper-data-collector = "^1.0.4"
+python-simple-logger = "^1.0.5"
+
+[tool.poetry.group.dev.dependencies]
+ipdb = "^0.13.13"
```

Comment:
Please add `ipython = "*"` (this cannot be add via poetry add command)

Reply:
Done